packages/circuits/test/QRVerify.test.ts

```diff
@@ -48,6 +51,7 @@ describe('Test QR Verify circuit', function () {
       message_len: messageLen,
       signature: splitToWords(data[1], BigInt(64), BigInt(32)),
       modulus: splitToWords(data[2], BigInt(64), BigInt(32)),
+      selector: [1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
```

Comment:
Let create document for selector.

Reply:
What do you mean?

Reply:
I mention about document for selector. How to help developer understand query data in QR by selector?

Reply:
Maybe we can abstract it by just having as the parameter of the pcd prover an array that you append with values you want to extract.
e.g. `reveal: ["age", "street"]`

Reply:
We have SelectorBuilder class, this help us to build selector easier. 
Example: new SelectorBuilder().selectEmailOrPhone().selectName().build() for select name and other field